shared/js/background/chrome-events.es6.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * NOTE: this needs to be the first listener that's added
+ *
+ * on FF, we might actually miss the onInstalled event
+ * if we do too much before adding it
+ */
+const ATB = require('./atb.es6')
+
+chrome.runtime.onInstalled.addListener(function(details) {
+    // only run the following section on install and on update
+    if (details.reason.match(/install|update/)) {
+        ATB.updateATBValues();
```

Comment:
Changed the method name from `onInstalled()`, so it's more descriptive. (Since there's other things that ATB could do when that event hits.)